app/web/package.json

```diff
@@ -22,41 +22,41 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@date-io/dayjs": "1.x",
-    "@emotion/cache": "^11.13.1",
     "@emotion/react": "^11.13.5",
-    "@emotion/server": "^11.11.0",
```

Comment:
Was an experiment when upgrading MUI that I forgot to remove.

Reply:
Interesting, it "sounds" like a package that'd be needed for server-side rendering but I trust that if things work, better to go without until we need it!

Reply:
@darrenvong we will need it for the next version of MUI but need to update to Next.js v13 and above first.